server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -362,7 +375,11 @@ export class TenantManager {
 			}
 
 			const encryptedTenantKey1 = tenantDocument.key;
-			const tenantKey1 = this.secretManager.decryptSecret(encryptedTenantKey1);
+			const encryptionKeyVersion = tenantDocument.customData?.encryptionKeyVersion;
```

Comment:
We need to handle the case where in encryptionKeyVersion is empty in cache/db.

Reply:
encryptionKeyVersion can be empty here. SecretManager, which is the only consumer of this value, is supposed to handle if encryptionKeyVersion is empty. Actually, before we run the script to encrypt tenant keys with new kek, it should always be empty. No issues found in testing when the value is empty.